src/commands/verify.ts

```diff
@@ -2,7 +2,7 @@ import { Argv } from "yargs";
 import signale from "signale";
 import { VerifyCommand } from "./command-types";
 import { isValid, VerificationFragment, verify } from "@govtechsg/oa-verify";
-import { readDocumentFile } from "../implementations/wrap/diskUtils";
+import { readDocumentFile } from "../implementations/diskUtils";
```

Comment:
I will suggest to move diskUtils to a folder called utils and rename the file disk (or anything better)
=> utils/disk